### DISCUSSION TITLE
EUla config

### DISCUSSION THREAD

**Discussion by jquintinoscript** [microsoft/winget-cli#1540] [General] [answered]
hi to all

i dont undertand how to config  to eliminate required user interaction.

User comands ou config on config file json? (-a, --accept-eula)

tKs
Jorge Quintino

**Comment by denelon** [comment, marked as answer]
Using `--accept-source-agreements` and `--accept-package-agreements` should do what you are looking for.

They are listed in `winget install -?` and [documented](https://docs.microsoft.com/windows/package-manager/winget/install).

**Reply by jquintinoscript** [reply]
thanks for response,
use to multi install in row 
 like this 
winget install --id=Adobe.AdobeAcrobatReaderDC -e -h --scope "machine" ; winget install --id=Microsoft.WindowsTerminal -e -h ; winget install --id=7zip.7zip -e -h --scope "machine" 

my questiosn is 

i must use one time or every instance to install 

tks

Jorge quintino

**Reply by denelon** [reply]
For source agreements, you should only have to accept once (until the agreements change).
For package agreements, you will need to accept for each one coming from the "**msstore**" source, and for each one coming from a private source with agreements per package.
